htdocs/compta/accounting-files.php

```diff
@@ -503,7 +504,7 @@
 $supplier_invoice = new FactureFournisseur($db);
 $expensereport = new ExpenseReport($db);
 $don = new Don($db);
-$salary_payment = new PaymentSalary($db);
+$salary_payment = new Salary($db);
```

Comment:
There is something strange here. The bank journal is bank of payment. So we will dispatch into accountancy payment.
So why is the object used here is Salary and not PaymentSalary ?

Reply:
You're right it was a mistake, i just corrected that